File: Jiu Jitsu Journal/Models/Class.cs
using static Azure.Core.HttpHeader;

namespace Jiu_Jitsu_Journal.Models
{
    public class BjjClass
    {
        public int Id { get; set; }
        public string Notes { get; set; }
        public string TypeOfClass { get; set; }
        public DateTime Date { get; set; }
        public UserProfile? UserProfile { get; set; }
        public int UserProfileId { get; set; }

        public int RollCount { get; set; }

    }
}

File: Jiu Jitsu Journal/Client/jiu-jitsu-journal/src/components/Class.js
import React from "react";
import { Card, CardBody } from "reactstrap";
import { Link } from "react-router-dom";

export const Class = ({ bjjClass }) => {
  return (
    <Card className="m-4">
      <CardBody>
        <p>
          <strong>Date: </strong>
          <Link to={`/class/${bjjClass.id}`}>
            {new Date(bjjClass.date).toLocaleDateString()}
          </Link>
        </p>
        <p>
          <strong>Notes: </strong>
          {bjjClass.notes}
        </p>
        <p>
          <strong>Type of Class: </strong>
          {bjjClass.typeOfClass}
        </p>
        <p>
          <strong>Posted by: </strong>
          {bjjClass.userProfile.fullName}
        </p>
        <p>
          <strong>Roll Count: </strong>
          {bjjClass.rollCount}
        </p>
      </CardBody>
    </Card>
  );
};
